test/print.jl

```diff
@@ -793,3 +793,132 @@ end
     @variable(model, z, base_name = "z^1")
     io_test(IJuliaMode, z, "z\\^1")
 end
+
+@testset "Print solution summary" begin
+
+    m = Model()
+    @variable(m, x <= 2.0)
+    @variable(m, y >= 0.0)
+    @objective(m, Min, -x)
+    c = @constraint(m, x + y <= 1) # anonymous constraint
+
+    JuMP.set_name(JuMP.UpperBoundRef(x), "xub")
+    JuMP.set_name(JuMP.LowerBoundRef(y), "ylb")
+
+    modelstring = """
+    variables: x, y
+    minobjective: -1.0*x
+    xub: x <= 2.0
+    ylb: y >= 0.0
+    noname: x + y <= 1.0
+    """
+
+    model = MOIU.Model{Float64}()
```

Comment:
It looks like this `model` variable is unused?

Reply:
Thanks, I removed it.